cmd/res/profiles/device.virtual.object.yaml

```diff
@@ -0,0 +1,13 @@
+name: "Fixed-Object-Device"
+manufacturer: "IOTech"
+model: "Device-Virtual-01"
+labels:
+  - "device-virtual-example"
+description: "Example of Device-Virtual"
+
+deviceResources:
+  - name: "Object"
```

Comment:
Shall we explicitly specify `isHidden` here, just like other virtual resources?

Reply:
added `isHidden` and `description`